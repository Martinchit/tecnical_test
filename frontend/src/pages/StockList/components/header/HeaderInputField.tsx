import styled from 'styled-components';
import { FormControl } from 'react-bootstrap';

export const HeaderInputField = styled(FormControl)`
  width: 30%;
  min-width: 250px;

  @media (max-width: 767px) {
    width: 100%;
  }
`;
