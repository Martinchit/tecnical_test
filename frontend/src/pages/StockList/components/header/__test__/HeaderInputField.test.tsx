import React from 'react';
import { shallow } from 'enzyme';

import { HeaderInputField } from '../HeaderInputField';

describe('HeaderInputField', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<HeaderInputField />);
    expect(wrapper).toMatchSnapshot();
  });
});
