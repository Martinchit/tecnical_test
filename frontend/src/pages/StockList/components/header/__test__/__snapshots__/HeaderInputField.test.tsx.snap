// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderInputField renders correctly 1`] = `
<FormControl
  className="sc-AxjAm hInynK"
/>
`;
